trafilatura/xpaths.py

```diff
@@ -36,7 +36,7 @@
     contains(@class, 'single-post') or
     contains(@class, 'main-column') or contains(@class, 'wpb_text_column') or
     starts-with(@id, 'primary') or starts-with(@class, 'article ') or @class="text" or
-    @class="cell" or @id="story" or @class="story" or
+    @class="cell" or @id="story" or @class="story" or contains(@class, "body-description") or
```

Comment:
Not necessary as per https://github.com/adbar/trafilatura/issues/112#issuecomment-919099605

Each expression slows things down a bit and there is a powerful algorithm working as fallback so I would refrain from adding to many constraints to `xpaths.py`.